tools/src/test/scala/caliban/tools/ClientWriterSpec.scala

```diff
@@ -510,21 +510,21 @@ object Client {
     case object NEWHOPE extends Episode
     case object EMPIRE  extends Episode
     case object JEDI    extends Episode
-    case object jedi_   extends Episode
+    case object `jedi_` extends Episode
```

Comment:
I wonder why there is still an underscore here, I would have expected `jedi`, no?

Reply:
Ah it comes from `getMappingsClashedNames` adding underscores at the end which then triggers your new logic.
How about changing `getMappingsClashedNames` to use backticks instead? I think it would be nicer.

Reply:
Didn't know the exact place where you haven't been happy with the test code. And I agree, a Jedi definitely looks better with backticks instead of backticks and underscore. ;-)